assets/section-email-signup-banner.css

```diff
@@ -33,3 +49,23 @@
 .email-signup-banner .banner__media {
   overflow: hidden;
 }
+
+@media screen and (max-width: 749px) {
+  .banner--mobile-content-align-left .newsletter-form__message {
+    justify-content: flex-start;
+  }
+
+  .banner--mobile-content-align-right .newsletter-form__message {
+    justify-content: right;
+  }
+}
+
+@media screen and (min-width: 750px) {
+  .banner--content-align-center .newsletter-form__message {
+    justify-content: center;
+  }
+
+  .banner--content-align-right .newsletter-form__message {
+    justify-content: right;
+  }
+}
```

Comment:
<img width="1098" alt="Screen Shot 2021-11-30 at 4 16 45 PM" src="https://user-images.githubusercontent.com/26778637/144129432-48487353-bc1d-43ff-b88e-b6a3fe726ffa.png">

<img width="1097" alt="Screen Shot 2021-11-30 at 4 18 40 PM" src="https://user-images.githubusercontent.com/26778637/144129676-db8d9af9-887e-4351-b15b-673ea882af32.png">

<img width="1096" alt="Screen Shot 2021-11-30 at 4 19 13 PM" src="https://user-images.githubusercontent.com/26778637/144129730-7eaa3477-210a-4275-ae55-c1c93581f541.png">


I wonder if we could remove padding top/bottom when the textbox is not visible. I just find the top/middle and middle/bottom positioning relatively similar when the padding is there.

Reply:
Personally I find that you can differentiate them pretty well 🤔 

What do you think @wiktoriaswiecicka @melissaperreault https://screenshot.click/01-04-oskf2-bk3vo.mp4 ?

Reply:
Yeah strange. I'd pulled in the latest changes too 🤷‍♀️ Anyways, it looks good now 👍🏻